include/mbgl/shaders/gl/drawable_fill_outline.hpp

```diff
@@ -11,7 +11,6 @@ struct ShaderSource<BuiltIn::FillOutlineShader, gfx::Backend::Type::OpenGL> {
     static constexpr const char* vertex = R"(layout (std140) uniform FillOutlineDrawableUBO {
     highp mat4 u_matrix;
     highp vec2 u_world;
-    highp vec2 pad;
```

Comment:
This is a needed padding field to keep the ubo size multiple of 16 bytes.

Reply:
Then I probably need to add it in the glsl file to in sync.